HISTORY.rst

```diff
@@ -1,8 +1,8 @@
 Release History
 ===============
 
-Unreleased Changes (2.0)
-------------------------
+Unreleased Changes
+------------------
```

Comment:
I think James added the two different `Unreleased` sections to better differentiate changes made. I guess it really doesn't matter because change remarks added for `release 2.0` won't make it into `master` but all the `master` change remarks will make it into the `release 2.0`. So it seems to me it's just to try and make it more human readable for us developers to keep of what changes are specific to this release and what changes will being going out beforehand in a bugfix release. Anyway, just wanted to make sure you meant to change this.

Reply:
Yes I meant to change it.